### PR TITLE
Fix wb links

### PR DESCRIPTION
**Pull request by nathandunn**
This fixes wormbase links.

Not sure how to run the tests, but I updated those as well.  The example links work correctly.

### CODE REVIEW

**Comment by nathandunn** [comment]
ping @kltm    @sibyl229 .   (not sure what Adam's is)

Fixes Wormbase links in reference to this: https://agr-jira.atlassian.net/browse/AGR-941?focusedCommentId=18087&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-18087 .

**Comment by sibyl229** [comment]
@nathandunn Looks good to me

File: golr/build_golr_schema.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+~/git/OWLTools-GO-svn/bin/owltools -solr-config ~/git/amigo/metadata/ont-config.yaml ~/git/amigo/metadata/bio-config.yaml ~/git/amigo/metadata/ann-config.yaml ~/git/amigo/metadata/ann_ev_agg-config.yaml -solr-schema-dump > ~/git/amigo/golr/solr/conf/2-schema.xml
```
Comment (kltm) [inline golr/build_golr_schema.sh]: What is this and why is it here?

**Comment by kltm** [comment]
@nathandunn Could you do this work on a branch so I can hop over to it without checkout out your namespace?

**Comment by kltm** [comment]
external/db-xrefs.yaml while it can (and should) be periodically updated to the latest upstream in `master`, it does not directly affect amigo2-instance-data builds, which, IIRC, are done with a pull of the current data directly.

**Comment by nathandunn** [comment]
@kltm

I need write access to amigo to do that. 

If you don’t want to do that, you should be able to create a branch and pull it in.   

Let me know work works.

**Comment by kltm** [comment]
If you can't run the build or the tests for amigo2-instance-data, you should make just a PR for the additional tests linked to a ticket in the AmiGO issue tracker and I can add them. (re: javascript/lib/amigo/linker.js.tests)

**Comment by kltm** [comment]
You're on the team now.

**Comment by nathandunn** [comment]
Alright.  You want me to push this onto a geneontology/amigo branch?

**Comment by nathandunn** [comment]
@kltm   I pushed it here as well:

https://github.com/geneontology/amigo/tree/fix-wb-links

**Comment by kltm** [comment]
I'm having trouble making the branch the source of the PR, versus this fork. Maybe this needs to be closed first?
The issue to reference is #492

**Comment by nathandunn** [comment]
@kltm  If you want to close this to refork that is fine.

**Pull request closed without merging**